src/editor.cpp

```diff
@@ -357,6 +357,35 @@ void Editor::addNewWildMonGroup(QWidget *window) {
     }
 }
 
+void Editor::deleteWildMonGroup() {
+    QComboBox *labelCombo = ui->comboBox_EncounterGroupLabel;
+
+    if (labelCombo->count() < 1) {
+        return;
+    }
+
+    QMessageBox msgBox;
+    msgBox.setText("Confirm Delete");
+    msgBox.setInformativeText("Are you sure you want to delete " + labelCombo->currentText() + "?");
+
+    QPushButton *deleteButton = msgBox.addButton("Delete", QMessageBox::DestructiveRole);
+    msgBox.addButton(QMessageBox::Cancel);
+    msgBox.setDefaultButton(QMessageBox::Cancel);
+    msgBox.exec();
+
+    if (msgBox.clickedButton() == deleteButton) {
+        auto it = project->wildMonData.find(map->constantName);
+        it.value().erase(labelCombo->currentText());
+
+        int i = project->encounterGroupLabels.indexOf(labelCombo->currentText());
```

Comment:
Do we need to be more defensive for this logic?  For example, is it possible for `it` to be null, or for `i` to be `-1`?  I just want to be sure this can't crash.

Reply:
Yeah you are right. Both have out of bounds values that I should've checked. Updated.